source/entries.d.ts

```diff
@@ -0,0 +1,85 @@
+// `Map` Helpers
+type MapKey<BaseType> = BaseType extends Map<infer KeyType, unknown> ? KeyType : never;
+type MapValue<BaseType> = BaseType extends Map<unknown, infer KeyType> ? KeyType : never;
```

Comment:
```suggestion
type MapValue<BaseType> = BaseType extends Map<unknown, infer ValueType> ? ValueType : never;
```

Reply:
thanks! fixed in https://github.com/sindresorhus/type-fest/pull/130/commits/1362c622d84f76445c2246389e671979dd8e65e9